requirements.txt

```diff
@@ -1,15 +1,9 @@
 # Avocado functional requirements
-# SSH lib (avocado.utils.remote)
-fabric>=1.11.0
 # multiplexer (avocado.core.tree)
 PyYAML>=3.11
```

Comment:
Now we need to make `varianter` optional plugin to get even slimmer (and sexier 😄)